app/src/main/res/layout/fragment_main.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+
+    <Button
+        android:id="@+id/btnNewGame"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="200dp"
+        android:onClick="newGame"
```

Comment:
Assign listeners in the fragment/activity.
This makes the XML layout reusable and lets you use simple lambdas instead of having to create specific functions named such and such.

Reply:
Same goes for other places like this.

Reply:
I just wanted to show you that I can do both ways, since in my fragment gameplay I did it the "proper" way with listener and lambda